src/tests/ftest/container/fill_destroy_loop.yaml

```diff
@@ -0,0 +1,17 @@
+hosts:
+  test_servers: 3
+server_config:
+  name: daos_server
+timeout: 500
+pool:
+  name: daos_server
+  control_method: dmg
+  scm_size: 1G
```

Comment:
Could we use `size` instead, and let DAOS figure out if it's SCM and/or NVMe based on the config? Or do we specifically only ever want SCM?